components/layout/featured.hbs

```diff
@@ -0,0 +1,51 @@
+<div class="section--bg-sec-blue">
+  <section class="featured">
+    <hgroup class="featured__title">
+      <h4 class="headline">Featured</h4>
+    </hgroup>
+    <div class="content-block--featured">
+      <div class="content-block__upper">
+        <img class="content-block__image" src="/img/featured_image.jpg" />
+      </div><!-- .content-block__upper -->
+
+      <div class="content-block__lower">
+        <p class="content-block__subtitle">2021 Salary Guide</p>
+        <h4 class="content-block__title">Unprecedented Challenges. Unexpected
+          Opportunities.</h4>
+
+        <p class="content-block__text">Lorem ipsum dolor sit amet, consectetur
+          adipiscing elit, sed do eiusmod tempor.</p>
+        <a href="#" class="button--medium-primary content-block__button">Learn
+          More</a>
+      </div><!-- .content-block__lower -->
+    </div><!-- .content-block--featured -->
```

Comment:
I'm not sure if this is helpful or not, but you might look at it: https://fractal.build/guide/components/including-sub-components.html

If we can include the already written pattern file here to prevent the chances of code drift between elements, we should. I know that this being a mod version of the base pattern might complicate this though.